app/app.py

```diff
@@ -261,6 +244,26 @@ async def startup(app: FastAPI):
         allow_headers=["*"],
     )
 
+    # Initialize database connection
+    app.dependency_overrides.get(get_db_engine, get_db_engine)(settings=settings)
```

Comment:
Is this useful? We don't use the returned engine.

Reply:
It creates the Async Engine, which is stored as a global variable and then used by all workers